app/controllers/manual_sections_controller.rb

```diff
@@ -1,3 +1,9 @@
 class ManualSectionsController < ApplicationController
+  def show
+    @section = Section.find(content_id: params[:content_id])
```

Comment:
I reckon a 404 would be better than a redirect with flash.  I don't think using `ActiveRecord::RecordNotFound` is the right thing to do though as the app isn't using ActiveRecord, so while we might get the right result (a 404) it'd be an unusual way to do it.  We could raise our own `ManualNotFound` error and add a `rescue_from` to the `ApplicationController` to make it more explicit.